src/transformStateWithClones.js

```diff
@@ -64,7 +64,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const newState = [];
+  const varState = { ...state };
+
+  for (const index of transforms) {
+    switch (index.operation) {
+      case 'addProperties' :
+        newState.push({ ...Object.assign(varState, index.properties) });
```

Comment:
```suggestion
        newState.push({ ...Object.assign(varState, index.properties)});
```

Reply:
Linter don't let

Reply:
it is correct sintaksis { ...obj }; 
For object literals (new in ECMAScript 2018):

let objClone = { ...obj };

Reply:
oh, yeah, sorry, didn't notice space on the other side